packages/lab/src/combo-box-next/useComboboxPortal.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  flip,
+  limitShift,
+  offset,
+  Placement,
+  shift,
+  size,
+  useDismiss,
+  useInteractions,
+  useRole,
+} from "@floating-ui/react";
+import { HTMLProps, useState } from "react";
+import { useFloatingUI } from "@salt-ds/core";
+
+export interface UsePortalProps {
+  open?: boolean;
+  onOpenChange?: (open: boolean) => void;
+  placement?: Placement;
+}
+
+export function useComboboxPortal(props?: UsePortalProps) {
```

Comment:
```suggestion
export function useComboBoxPortal(props?: UsePortalProps) {
```
would rename file, and rename props to UseComboBoxPortalProps